src/main/java/com/rultor/web/TkAppFallback.java

```diff
@@ -105,7 +104,7 @@ private static Response fatal(final RqFallback req) throws IOException {
                     TkAppFallback.class.getResource("error.html.vm"),
                     new RsVelocity.Pair(
                         "err",
-                        ExceptionUtils.getStackTrace(req.throwable())
+                        req.throwable().getStackTrace()
```

Comment:
@pnatashap I believe, this is not exactly what `ExceptionUtils.getStackTrace` does. We should use `Logger.format("%[exception]s", req.throwable())` instead.